docs/operator.md

```diff
@@ -38,7 +38,6 @@ The spec part of the [deploy/cr.yaml](https://github.com/percona/percona-server-
 | replsets        | [subdoc](operator.md#operator-replsets-section)       | | Operator MongoDB Replica Set section |
 | pmm             | [subdoc](operator.md#operator-pmm-section)            | | Percona Monitoring and Management section |
 | sharding        | [subdoc](operator.md#operator-sharding-section)       | | MongoDB sharding configuration section |
-| mongod          | [subdoc](operator.md#operator-mongod-section)         | | Operator MongoDB Mongod configuration section |
```

Comment:
Actually it seems we forgot to remove this section from 1.14.0 so we will need to do it in the next release. We need to change the docs to say we will remove it in 1.15.0.